### PR TITLE
Add demo link.

### PR DESCRIPTION
**Pull request by lewei50**
## Proposed change
Add demo link.

## Type of change
- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by lewei50** [comment]
What should I do to make this PR continue ?

**Comment by frenck** [comment]
I've looked at the demo and it looks cool. But it exposes IP addresses of anyone who as checked it out, viewable by anybody. This is a privacy issue.

**Comment by frenck** [comment]
I think, as an alternative approach, a screenshot can be added.

**Comment by lewei50** [comment]
This test account has guest rights only,it can view and operate the light to see the power meter changes only.
The demo environment is isolated,so there is no risk to expose any devices's IP.
Operating the real lights will give guest better experience in HA I think.

**Comment by frenck** [comment]
@lewei50 Yes there is, as you can view all access tokens in the user account and thus see all IP addresses of people logged into the demo.

Please use a screenshot instead.

**Comment by lewei50** [comment]
> @lewei50 Yes there is, as you can view all access tokens in the user account and thus see all IP addresses of people logged into the demo.
> 
> Please use a screenshot instead.

Thanks for your expaination.I'll attach a screenshot .

**Comment by lewei50** [comment]
I have replaced the url to a snapshot.
Thanks a lot.

File: source/_integrations/iammeter.markdown [hunk 1]
```diff
@@ -16,11 +16,8 @@ ha_codeowners:
   
 demo show:
 
-[http://ha.iammeter.com:18123](http://ha.iammeter.com:18123)
+![image-20200623101938530](https://leweidoc.oss-cn-hangzhou.aliyuncs.com/lewei50/img/iammeter/iammeterimage-20200623101938530.png)
```
Comment (frenck) [inline source/_integrations/iammeter.markdown]: Screenshots must be included in this PR.

**Comment by lewei50** [comment]
I have uploaded the snapshot to ../images/snapshots/ directory and reedited the markdown file.
wish it works.

File: source/_integrations/iammeter.markdown (hunk 1 above)
Comment (lewei50) [reply source/_integrations/iammeter.markdown]: > Screenshots must be included in this PR.

Would you help to check if I did it right this time?

Thanks!

File: source/_integrations/iammeter.markdown [hunk 2]
```diff
@@ -13,6 +13,14 @@ ha_codeowners:
 
 `iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
 
+  
+demo show:
```
Comment (zsarnett) [inline source/_integrations/iammeter.markdown]:
```suggestion
Example Lovelace Dashboard:
```

`demo show:` is not great wording in my opinion. Let's describe it a bit more?

File: source/_integrations/iammeter.markdown [hunk 3]
```diff
@@ -13,6 +13,14 @@ ha_codeowners:
...
+
+<p class='img'>
+  <img src='{{site_root}}/images/screenshots/iammeter demo.jpg' />
```
Comment (zsarnett) [inline source/_integrations/iammeter.markdown]: do you need this `{{site_root}}` here? 

Also, Can you rename the file to be with out spaces?

Comment (lewei50) [reply source/_integrations/iammeter.markdown]: {{site_root}} removed and filename renamed.

thanks for your answer.

**Review by frenck — APPROVED** [review]

**Pull request merged**